rakelib/update_bundle_audit.rake

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
+require 'rubygems'
+require 'bundler/setup'
+
 desc 'Update definitions used in bundle-audit'
 task :update_bundle_audit do
-  sh 'bundle-audit update'
+  sh 'echo $PATH' # TODO
+  sh 'env' # TODO
```

Comment:
[punchlist] Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker